README.md

```diff
@@ -35,7 +69,7 @@ Bulk Inserting ``1,000,000``entities to a SQL Server 2016 database took ``5`` Se
 The domain model could be the ``Person`` class with a First Name, Last Name and a birth date. 
 
 ```java
-// Copyright (c) Philipp Wagner. All rights reserved.
+// Copyright (c) Philipp Wagner and Victor Lee. All rights reserved.
```

Comment:
Do we also have to put the copyrights into the README.md? Since these code examples go into ppl's personal projects, they may not want to have to copy the copyrights into their projects.

Reply:
Point for you! I removed the copyrights.